src/components/Budget/BudgetView/BudgetSummary.js

```diff
@@ -21,6 +21,17 @@ const BudgetSummary = ({ budget, fiscalYearCurrency }) => (
         />
       </KeyValue>
     </Col>
+    <Col xs={3}>
+      <KeyValue
+        label={<FormattedMessage id="ui-finance.budget.netTransfers" />}
+      >
+        <AmountWithCurrencyField
+          amount={budget.netTransfers}
+          currency={fiscalYearCurrency}
+          showBrackets
```

Comment:
IMHO, this should be like `showBrackets={budget.netTransfers < 0}`

Reply:
🤦‍♂️ sure